### DISCUSSION TITLE
Water on Rescueland

### DISCUSSION THREAD

**Discussion by badguyland** [hermonochy/Rescueland#24] [Q&A] [answered]
Do any water bodies exist on Rescueland? If not, how do the animals survive there?

**Comment by hermonochy** [comment, marked as answer]
Water does exist on Rescueland, although in lesser quantities than on Earth. It primarily exists as water vapor, with sporadic lakes scattered across solid land. The animals have adapted by either extracting moisture from the air, staying within the proximity of these lakes, or finding alternative sources of fluids.
